### PR TITLE
Fix unknown CMP0148 with a cmake minimum range

### PR DESCRIPTION
**Pull request by Ryanf55**
Solves #91

### CODE REVIEW

File: mav_msgs/CMakeLists.txt [hunk 1]
```diff
@@ -1,8 +1,6 @@
-cmake_minimum_required(VERSION 3.14.4)
+cmake_minimum_required(VERSION 3.14.4..3.27)
```
Comment (srmainwaring) [inline mav_msgs/CMakeLists.txt]: Should this use 3 dots `...`? ( i.e. `cmake_minimum_required(VERSION 3.14.4...3.27)`)

https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html

File: mav_msgs/CMakeLists.txt [hunk 2]
```diff
@@ -1,8 +1,6 @@
...
 project(mav_msgs)
 
-cmake_policy(SET CMP0148 OLD)
```
Comment (srmainwaring) [inline mav_msgs/CMakeLists.txt:4]: Setting the `max` on macOS (cmake version 3.27.7) with the 3 dots there is now an error:

```bash
CMake Error at /Users/rhys/Code/ros2/humble/ros2-humble/install/rosidl_generator_py/share/rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake:20 (find_package):
  By not providing "FindPythonInterp.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "PythonInterp", but CMake did not find one.

  Could not find a package configuration file provided by "PythonInterp"
  (requested version 3.6) with any of the following names:

    PythonInterpConfig.cmake
    pythoninterp-config.cmake

  Add the installation prefix of "PythonInterp" to CMAKE_PREFIX_PATH or set
  "PythonInterp_DIR" to a directory containing one of the above files.  If
  "PythonInterp" provides a separate development package or SDK, be sure it
  has been installed.
Call Stack (most recent call first):
  /Users/rhys/Code/ros2/humble/ros2-humble/install/ament_cmake_core/share/ament_cmake_core/cmake/core/ament_execute_extensions.cmake:48 (include)
  /Users/rhys/Code/ros2/humble/ros2-humble/install/rosidl_cmake/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:286 (ament_execute_extensions)
  CMakeLists.txt:16 (rosidl_generate_interfaces)


make: *** [cmake_check_build_system] Error 1
---
Failed   <<< mav_planning_msgs [5.77s, exited with code 2]```
```

We may also need:

```cmake
if(${CMAKE_VERSION} VERSION_GREATER 3.27)
  cmake_policy(SET CMP0148 OLD)
endif()
```

**Review by srmainwaring — CHANGES_REQUESTED** [review]

File: mav_msgs/CMakeLists.txt (hunk 1 above)
Comment (Ryanf55) [reply mav_msgs/CMakeLists.txt]: Yes!

File: mav_msgs/CMakeLists.txt (hunk 2 above)
Comment (Ryanf55) [reply mav_msgs/CMakeLists.txt:4]: Added the above. Please retest at your convenience.

**Review by srmainwaring — APPROVED** [review]
LGTM. Thanks for the fix.

**Pull request merged**